core/src/test/java/kafka/admin/ConfigCommandIntegrationTest.java

```diff
@@ -82,202 +105,448 @@ public void testExitWithNonZeroStatusOnUpdatingUnallowedConfig() {
             "--alter",
             "--add-config", "security.inter.broker.protocol=PLAINTEXT")),
             errOut ->
-                assertTrue(errOut.contains("Cannot update these configs dynamically: Set(security.inter.broker.protocol)"), errOut));
```

Comment:
Please revert this unrelated change. BTW, this test case should run all cluster types, right?

Reply:
True, this test can run at all type cluster tests